ReText/window.py

```diff
@@ -826,6 +826,7 @@ def openFile(self):
 		for fileName in fileNames[0]:
 			self.openFileWrapper(fileName)
 
+	@pyqtSlot(str)
```

Comment:
With this is possible to expose other methods of the UI on dbus, in case there are other needs